.github/workflows/test-agent.yml

```diff
@@ -248,7 +248,7 @@ jobs:
           LIBC: ${{matrix.platform}}
           PLATFORM: linux/${{matrix.arch}}
           AGENT_CODE: ${{github.workspace}}/php-agent
-          IMAGE_VERSION: ${{vars.MAKE_PHP_VERSION}}
+          IMAGE_VERSION: v2
```

Comment:
Is this a temporary change - not clear why this is being hard-coded.

Reply:
This workflow needs to change anyway in order to support running integration tests on arm64 and be more like run-integration-tests workflow, so I'm actually fine either way.

Reply:
This is necessary to get the PR runner to use the correct builder image. Changing this on a repo level would break all other PRs. Once this is merged another PR will be put up to revert the change.

Reply:
I agree with @bduranleau-nr assessment.